packages/ses/docs/lockdown.md

```diff
@@ -166,17 +196,34 @@ lockdown({
   overrideTaming: 'min', // Until https://github.com/endojs/endo/issues/636
 });
 ```
+
+If `lockdown` receives no election for `consoleTaming`, it will respect
+`process.env.LOCKDOWN_CONSOLE_TAMING`.
+
+```console
+LOCKDOWN_CONSOLE_TAMING=safe
+LOCKDOWN_CONSOLE_TAMING=unsafe
+```
+
+The `consoleTaming: 'safe'` setting replaces the global console with a tamed
+console, and that tamed console is safe to endow to a guest `Compartment`.
+Additionally, any errors created with the `assert` function or methods on its
+namespace may have redacted details: information included in the error message
+that is informative to a debugger and made invisible to an attacker.
+The tamed console removes redactions and shows these details to the original
+console.
```

Comment:
New paragraph is great and was badly needed. Please also add a link to https://github.com/endojs/endo/blob/master/packages/ses/src/error/README.md